app/[locale]/PartnersSponsors.tsx

```diff
@@ -0,0 +1,217 @@
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://ncloud.com"
+            target="_blank"
+          >
+            <Image
+              alt={t("navercloud")}
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height="40"
+              src="/images/sponsors-partners/navercloud.svg"
+              width="160"
+            />
+          </Link>
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://opendrives.com"
+            target="_blank"
+          >
+            <Image
+              alt={t("opendrives")}
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height="40"
+              src="/images/sponsors-partners/opendrives.png"
+              width="160"
+            />
+          </Link>
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://osuosl.org"
+            target="_blank"
+          >
+            <OslLogo
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height={40}
+              width={160}
+            />
+            <p className="sr-only">{t("osuosl")}</p>
+          </Link>
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://symphony.rakuten.com"
+            target="_blank"
+          >
+            <Image
+              alt={t("rakutensymphony")}
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height="40"
+              src="/images/sponsors-partners/rakutensymphony.svg"
+              width="160"
+            />
+          </Link>
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://supermicro.com"
+            target="_blank"
+          >
+            <Image
+              alt={t("supermicro")}
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height="40"
+              src="/images/sponsors-partners/supermicro.svg"
+              width="160"
+            />
+          </Link>
+          <Link
+            className="mx-auto flex w-full items-center justify-center p-4 sm:p-8 rounded-lg border"
+            href="https://vmware.com"
+            target="_blank"
+          >
+            <VmwareLogo
+              className="aspect-[4/1] overflow-hidden rounded-lg object-contain object-center"
+              height={40}
+              width={160}
+            />
+            <p className="sr-only">{t("vmware")}</p>
+          </Link>
+        </div>
```

Comment:
Fixed. I also took the liberty of organizing the sponsors and partners list into a single data helper since we'll be re-using this in different components on a few more pages.